classes/Game.js

```diff
@@ -0,0 +1,268 @@
+const Player = require('./Player.js')
+const NetCode = require('./NetCode.js').NetCode
+const accurateInterval = require('accurate-interval')
+
+const DELTATIME = 0.01 // Matches FixedUpdate on client side
+const GRAVITY = -9.81 * 1.5
+const GROUND = 0
+const A_BIT = 0.05
+
+class Game {
+    constructor(serverSocket, process) {
+        // Pools of game entities
+        this.players = []
+        this.blocks = []
+
+        // Set server details
+        this.server = serverSocket;
+        this.process = process;
+
+        // Setup physics loop
+        accurateInterval(() => {
+            this.PhysicsTick()
+        }, DELTATIME * 1000, {
+            aligned: true,
+            immediate: true
+        })
+
+        // Check for players who disconnects
+        let interval = 10 * 1000
+        setInterval(() => {
+            let time = Date.now()
+            for (let i = 0; i < this.players.length; i++) {
+                if (time - this.players[i].lastseen >= interval) {
+                    NetCode.Do("Disconnect", [
+                        this.players[i].uid
```

Comment:
I think this would look better inline. You've done single parameters inline elsewhere, anyways.